src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -321,6 +354,7 @@ public static IEnumerable<object[]> UseCallback_ValidCertificate_ExpectedValuesD
             }
         }
 
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "UAP doesn't expose channel binding information")]
```

Comment:
Yep.  I mean, the function does *something*.  Whether that be return null, throw, give a fixed value of the integer digits of pi... whatever it does we should validate that it's doing it so that a) we know if it changes (may have doc impact, a bunch of tests that need to be enabled now, etc) and b) if a customer opens an issue it's much easier to say "by design" if we can point at an existing test.